app/services/csv_export/match_exporter.rb

```diff
@@ -19,8 +19,10 @@ def fields
         company_categorie_juridique: -> { company.categorie_juridique },
         company_naf: -> { facility.naf_code },
         company_effectif: -> { Effectif::CodeEffectif.new(facility.code_effectif).intitule_effectif },
+        company_forme_exercice: -> { company&.forme_exercice },
         inscrit_rcs: -> { company.inscrit_rcs ? I18n.t('boolean.text.true') : I18n.t('boolean.text.false') },
         inscrit_rm: -> { company.inscrit_rm ? I18n.t('boolean.text.true') : I18n.t('boolean.text.false') },
+        activite_liberale: -> { company.activite_liberale ? I18n.t('boolean.text.true') : I18n.t('boolean.text.false') },
```

Comment:
on devrait pouvoir faire un truc comme 
```suggestion
        activite_liberale: -> { I18n.t(company.activite_liberale, scope: [:boolean, :text]) },
```

Reply:
bin oui, tout à fait
("pourquoi faire simple....")